packages/fuels-accounts/src/provider/retryable_client.rs

```diff
@@ -17,10 +17,12 @@ use fuels_core::types::errors::{error, Error, Result};
 
 use crate::provider::{retry_util, RetryConfig};
 
+use super::supported_versions::{self, VersionCompatibility};
+
 #[derive(Debug, thiserror::Error)]
 pub(crate) enum RequestError {
-    #[error(transparent)]
-    IO(#[from] io::Error),
+    #[error("io error: {0}")]
```

Comment:
Small nit. I would not add `io error:` here. All errors coming from the provider will now have `io::error` prepended to the actual error message like this: `provider: io error: <some error>`

Reply:
Makes sense. Can we make this a follow up? I dislike this being part of the error at all. I'd rather make it a `log::warn` but we don't log anything else and we should probably start and document that we do and that users should enable logging in their apps in order to see the info. Maybe even expose the version compat checking fn so that users can still make this into a hard error if they want.

Reply:
Sounds good to me.